src/main/resources/db/migration/V228__Add_AccountNumber_to_LineItems.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `LineItems` CHANGE COLUMN `AccountNumber` `DocumentNumber` VARCHAR(50) NULL;
```

Comment:
Should rename this migration to a more accurate description

Reply:
Fixed now. Forgot to change the name when copying the file.